workflows/images/cellranger/README.md

```diff
@@ -0,0 +1,39 @@
+This folder contains a Dockerfile for the cellranger analysis.
+
+## Building the image
+
+In order to build this image, the cellranger archive component must be downloaded separately to comply with licensing, and should be placed in this folder.
+The current version of this file is `cellranger-4.0.0.tar.gz` and can be downloaded from [10X Genomics Website](https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest) after agreeing to their license terms.
+
+Following download of cellranger, you can build the image with:
+```
+docker build . -t scpca-cellranger:4.0.0
+```
+
+At this point, the image should be ready for use on the local machine.
+
+## Pushing to AWS ECR
+If the updated image needs to be pushed to AWS ECR, you can follow the steps below to push to the Alex's Lemonade ECR repository.
+
+First, login to the AWS ECR docker repository with:
+```
+aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 589864003899.dkr.ecr.us-east-1.amazonaws.com
+```
+Note that for the code above to work, you must have set up `aws` command line tools on your machine, and have run [`aws configure`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) to set up your credentials for AWS access.
```

Comment:
We don't want to run this in a docker image? I don't have aws installed so I'll have to do this. Seems like this would still be handy to have in a docker image too, but it seems you have a lot of docker images going on?

Reply:
Does it matter if I get version 1 or version 2? https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html

Reply:
No. This is infrastructure that most people don't need to set up or reproduce. Not doing dockers inside dockers for people to use AWS.

Reply:
Does it matter if its version 1 or 2 though?

Reply:
I don't think it does. I apparently have v1, which is what was available through `conda`.
Some investigation shows that `aws ecr get-login-password` is not available on versions below 1.17, but that is pretty old.